test/index.test.js

```diff
@@ -10,11 +10,17 @@ const tests = fs.readdirSync(__dirname).filter(file => {
 
 tests.forEach(async (test) => {
```

Comment:
Can we remove the `async` directive from here now? CI test failed on Node 4/6.

Reply:
Whooops. Thought I removed them all. Dont need that one!

Reply:
It's a shame the tests take so long to run!